packages/baseten/src/baseten-provider.integration.test.ts

```diff
@@ -0,0 +1,287 @@
+import { describe, it, expect, beforeAll } from 'vitest';
+import { baseten, createBaseten } from '@ai-sdk/baseten';
+import { embed, embedMany, generateText } from '../../ai';
+
+// ============================================================================
+// BASETEN EMBEDDING PROVIDER INTEGRATION TESTS
+// ============================================================================
+// 
+// This test suite validates the Baseten embedding provider with Performance Client.
+// 
+// Endpoint Types:
+// - /sync: OpenAI-compatible endpoints (Performance Client adds /v1/embeddings)
+// - /predict: Custom Baseten endpoints (Performance Client uses URL as-is)
+// 
+// Performance Client Configuration:
+// - Max concurrent requests: 128
+// - Batch size: 16
+// - Timeout: 720 seconds
+// 
+// ============================================================================
+
+// ============================================================================
+// CONFIGURATION - Update these URLs with your own model endpoints to run the tests
+// ============================================================================
+
+// Embedding model URLs - Performance Client with different endpoint types
+const EMBEDDING_SYNC_URL =
```

Comment:
what account are these models deployed on? Should we templatize this more so anyone could hypothetically run these? Are these meant to run in some kind of CI flow?